tests/test_integ_test_behave.py

```diff
@@ -52,7 +64,6 @@ def _build_image(overrides=None):
     return _build_image
 
 
-@pytest.mark.skipif(platform.system() == "Darwin", reason="Disabled on macOS")
```

Comment:
These tests seem to work find on macOS, not sure why the were disabled.

Reply:
Good question ; I'll have to look into that (although I don't have a Mac) ; it was changed in 1477fa22e9d9257e50096c208ee77f37838709bb